Assets/Scripts/Controllers/KeyboardController.cs

```diff
@@ -26,7 +25,6 @@ public class KeyboardController
     // Use this for initialization.
     public KeyboardController(BuildModeController buildModeController, WorldController worldController)
```

Comment:
Odd that there isn't a StyleCop warning for an unused parameter in a non-override or interface function.